### DISCUSSION TITLE
ADLX and GPU Fan stop working

### DISCUSSION THREAD

**Discussion by Rattow** [Rem0o/FanControl.Releases#2610] [General]
Hey guys, when i open the program show the message "ADLX could not initialize or has no sensors" and my GPU FUN is 0 temp and RPM. Any help please?

Version 196 (NET 4.8)

_sorry about my english, its not my main langague_

![imagem_2024-07-12_200426538](https://github.com/user-attachments/assets/80323b5f-e70c-4054-b01a-12ccfa072310)
![imagem_2024-07-12_200438020](https://github.com/user-attachments/assets/aeee764b-3514-48e7-a1c9-61d1116dc47b)

**Comment by Rem0o** [comment]
Make sure to disable the GPU section in sensor settings ( Librehardware column ).

Do you have access to this ?

**Reply by Rem0o** [reply]
![image](https://github.com/user-attachments/assets/0eef7994-63c5-4e68-b942-8c221394b817)

**Comment by Rattow** [comment]
Hey Rem0o thanks for the reply.

I really do know what is Librehardware column that you said, sorry 
edit: i found ^^
![image](https://github.com/user-attachments/assets/ca53dc1f-ecb5-4b96-aa14-973916a562a0)

Do you mean like this? 
![image](https://github.com/user-attachments/assets/aa369d30-cba8-4f29-ac42-9f5d5dc22b86)

**Reply by Rem0o** [reply]
Untick GPU on the left, tick ADLX on the right, untick the plugin.

**Reply by Rattow** [reply]
Like this, right? 
![image](https://github.com/user-attachments/assets/ca7968a0-aafa-4aa0-953a-894891a54902)

**Reply by Rem0o** [reply]
Exactly

**Reply by Rattow** [reply]
> Exactly

Thanks man! i think its working as intended
